src/models/base.spec.js

```diff
@@ -1,12 +1,30 @@
-/* global beforeEach, describe, expect, it */
 'use strict';
 
 import {Model} from './base';
+import {isString} from '../validators';
 
-QUnit.module('my example tests');
 
-QUnit.test('Model', function(assert) {
-     var expected = 'Hello Foo';
-	 var itb = new Model();
-     assert.ok(true);
+QUnit.module('Model');
+
+QUnit.test('Model constructor', function(assert) {
+     var model = new Model();
+
+     assert.ok(model !== null);
+});
+
+QUnit.test('Model extendValidation, set, and get', function(assert) {
+     var model = new Model();
+	 var validators = {'testKey': isString};
+
+	 model.extendValidation(validators);
```

Comment:
this will also need to be updated per @kenkouot's latest changes (not merged yet)